Json/class/Warlock_1 copy.json

```diff
@@ -0,0 +1,88 @@
+{
+  "ClassName": "Warlock",
+  "Loot": true,
+  "NPCMaxLevels_Below": 3,
+  "PathFilename": "test.json",
+  "PathThereAndBack": true,
+  "PathReduceSteps": false,
```

Comment:
this class profile has no additional info compare to the `Warlock_1.json`. 
Please delete this.